File: cfgs/pythoncfgs/PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES.py
Nodes
1;statement;foo_1 = 2 ** foo_2 - 1;8;
2;statement;foo_1 = 4 % foo_2;9;
3;conditional;foo < foo;10;
4;conditional;foo_1 == 0;12;
5;statement;return false;13;
6;statement;return true;12;
7;statement;foo_1 = (foo_2 * foo_3 - 2) % foo_4;11;
8;exit;;
Edges
0;2;
1;3;
2;4;7;
3;5;6;
4;8;
5;8;
6;3;
7;

File: cfgs/java_cfgs_foo/PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES.java
Nodes
1;special;;;
2;statement;double checkNumber = fun_1(2, foo_1) - 1;14;
3;statement;double nextval = 4 % foo_1;15;
4;statement;int i = 1;16;
5;conditional;foo_1 < (foo_2 - 1);17;
6;statement;nextval = ((foo_1 * foo_2) - 2) % foo_3;18;
7;statement;i++;18;
8;statement;return foo_1 == 0;19;
9;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;8;
5;7;
6;5;
7;9;
8;
